### ISSUE TITLE
Standard setting output-language

### ISSUE THREAD

**Issue by Somebodyisnobody** [Silbad/DeepLT#5]
My suggestion:
Input language standard -> Setting in Addonpage to choose between "Save last used language", "Save last used language but reset to [chose language here] when browser restarts"
Output language standard -> Setting in Addonpage to choose between "Use OS/Browser-language" and "Save last used language", "Save last used language but reset to [chose language here] when browser restarts". (The "don't save" option at the moment is nonsense, because in this case French is automaticlly set.)
Exemption: If input language = output language -> output-language = english

**Comment by YoRHa-A5** [comment]
Yes please, I would love that, too!

**Comment by rdmtinez** [comment]
Yes please! That would be great, especially if you're not living in France or studying French

**Comment by Kulhanekju** [comment]
Please add this functionality, it is very annoying to change the target language every time I use it.
